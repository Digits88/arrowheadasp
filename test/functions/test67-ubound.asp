<!-- Test UBound function.
-->

Dim A1(2), A2(3, 4), A3(5, 6, 7)
	<% Dim A1(2), A2(3, 4), A3(5, 6, 7) %><br>

UBound(A1) = <%= UBound(A1) %><br>
UBound(A2) = <%= UBound(A2) %><br>
UBound(A3) = <%= UBound(A3) %><br>
UBound(A1, 1) = <%= UBound(A1, 1) %><br>
UBound(A2, 1) = <%= UBound(A2, 1) %><br>
UBound(A3, 1) = <%= UBound(A3, 1) %><br>
UBound(A2, 2) = <%= UBound(A2, 2) %><br>
UBound(A3, 2) = <%= UBound(A3, 2) %><br>
UBound(A3, 3) = <%= UBound(A3, 3) %><br>
