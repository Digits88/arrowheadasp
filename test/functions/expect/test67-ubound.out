<!-- Test UBound function.
-->

Dim A1(2), A2(3, 4), A3(5, 6, 7)
	<br>

UBound(A1) = 2<br>
UBound(A2) = 3<br>
UBound(A3) = 5<br>
UBound(A1, 1) = 2<br>
UBound(A2, 1) = 3<br>
UBound(A3, 1) = 5<br>
UBound(A2, 2) = 4<br>
UBound(A3, 2) = 6<br>
UBound(A3, 3) = 7<br>
